FWCore/ParameterSet/src/FileInPath.cc

```diff
@@ -15,9 +16,12 @@ namespace bf = boost::filesystem;
 
 namespace
 {
+
+  std::atomic<bool> fileLookupDisabled{false};
```

Comment:
How about the name `s_fileLookupDisabled` since I have traditionally used `s_` for "global" quantities.

Reply:
done, that is better. Thanks.